routes/orders-routes.js

```diff
@@ -20,21 +15,21 @@ module.exports = (database) => {
       })
       .catch((err) => {
         return err.messages;
-      })
-  })
+      });
+  });
   // GET Orders / id
   router.get('/:id', (req, res) => {
 
-      if (req.params.id) {
+    if (req.params.id) {
       getOrderById(req.params.id)
-      .then((order) => {
-        res.send(order);
-      })
-      .catch((err) => {
-        res.send(err.messages);
-      })
+        .then((order) => {
+          res.send(order);
+        })
+        .catch((err) => {
+          res.send(err.messages);
```

Comment:
If there is an error, it will send the error message back to the user browser. Is. this the expected behaviour?